client/client_test.go

```diff
@@ -1976,6 +1977,20 @@ func testBuildMultiMount(t *testing.T, sb integration.Sandbox) {
 	checkAllReleasable(t, c, sb, true)
 }
 
+func testBuildScratch(t *testing.T, sb integration.Sandbox) {
```

Comment:
I think this test would need to export an image to catch the code path that is being fixed.

The test should fail before the patch and pass with the patch.